vmdb/app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -49,6 +49,18 @@ def show_container(record, controller_name, display_name)
         pluralize(@view.extras[:total_count] - @view.extras[:auth_count], "other #{title.singularize}")
         + " on this " + ui_lookup(:tables => @table_name)
       end
+    elsif @display == "container_services" || session[:display] == "container_services" && params[:display].nil?
+      title = ui_lookup(:tables => "container_services")
+      drop_breadcrumb(:name => record.name + " (All #{title})",
+                      :url  => "/#{controller_name}/show/#{record.id}?display=#{@display}")
+      @view, @pages = get_view(ContainerService, :parent => record)  # Get the records (into a view) and the paginator
+      @showtype = @display
+      if @view.extras[:total_count] > @view.extras[:auth_count] && @view.extras[:total_count] &&
+         @view.extras[:auth_count]
+        @bottom_msg = "* You are not authorized to view " +
+        pluralize(@view.extras[:total_count] - @view.extras[:auth_count], "other #{title.singularize}")
+        + " on this " + ui_lookup(:tables => @table_name)
```

Comment:
> @zeari can you address comment from @fryguy, going forward we are changing code to use interpolation. just make change to newly added code for this PR

Alright. Should I also apply the change with the self links?
```Ruby  
 = link_to(_('Container Services (0)'), "#")
```
> This should probably use the existing (_("Container Services (%s)") % num_services format instead of introducing an extra string that needs internationalization.